plugins/user-settings/src/components/General/UserSettingsGeneral.tsx

```diff
@@ -31,6 +32,9 @@ export const UserSettingsGeneral = () => {
       <Grid item xs={12} md={6}>
         <UserSettingsIdentityCard />
       </Grid>
+      <Grid item xs={12} md={6}>
+        <UserSettingsAvailableIconsTable />
```

Comment:
it could be cool to have this as part of a view similar to the scaffolder available actions view so that users can directly see in Backstage what is available for them to use and is also always up to date with what's installed in that particular instance

Reply:
@benjdlambert so the idea would be people could add this to their Advanced settings tabs? Sorry, not totally sure I follow

@kuangp yes, that was my idea with this, as teams have been adding their entities I get asked what icons are available for their use. This would do exactly as you suggest I'm just not sure the right "home" for it

Reply:
Sorry, maybe I'm thinking a little bit to future here, but it would be great if we could supply some from of storybook for `create-app`'s that can surface this stuff. Similar to the `/actions` page of the scaffolder you could hit `/storybook` of your backstage instance and see things like this that are automatically generated for you and are your building blocks from your company DLS. Just not really sure where else to put something like this, a card doesn't feel like the right solution IMO

Reply:
Ok, would it make sense to take out the card part and the changes to user-settings? That would at least get the docs updated and some of the pluming in place for this. Then what I'm thinking is maybe I create a `\contib` guide on how to build this yourself. I've been wanting to do something like that for how to create your own icons so I can do them together. How does that sound @benjdlambert?

Reply:
Hey, yeah so I like that approach, let's keep things a little separate from there for a start. I think that the additions for `getSystemIcons` is OK, but would really want to run it past some other maintainers first. Although it's not a major addition, just don't want to get too many people to rely on it for us to change it when the maintainers get back :joy: @backstage/maintainers @rugvip

Reply:
For sure, this is not a rush at all. I do think it would be handy for adopters to have this as not everyone (thinking various personas here) will be comfortable diving into the repo to figure out what icons they can use for their links

Reply:
Ok, cool, I've made those changes, hope it's alright but I also updated the title and description of this PR to better reflect the actual changes same with the changeset.

If I find some spare time I might add the `\contrib` tutorials to this PR